pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/search/ElasticSearchParticipantDto.java

```diff
@@ -72,7 +72,7 @@ protected ElasticSearchParticipantDto() {  }
      * and question do not exist.
      */
     @VisibleForTesting
-    public boolean changeQuestionAnswer(String activityCode, String questionStableId, String value) {
+    public boolean changeQuestionAnswer(String activityCode, String questionStableId, Object value) {
```

Comment:
I expanded this to `Object` because the underlying ES data will have these as booleans, and I wanted more realistic data here for testing.